### PR TITLE
Remove Facebook copyrights, update licenses to MIT

### PR DESCRIPTION
**Pull request by acao**
- remove copyright statements from all files
- update license files to MIT
- export LICENSE files from all projects
- update/add package.json license entries
- normalize import/require formatting across all files

### CODE REVIEW

File: packages/graphql-language-service-parser/src/RuleHelpers.js [hunk 1]
```diff
@@ -1,16 +1,10 @@
...
- *  LICENSE file in the root directory of this source tree.
- *
  *  @flow
  */
 
-// These functions help build matching rules for ParseRules.
+import type { Rule, Token } from 'graphql-language-service-types'
 
-import type { Rule, Token } from 'graphql-language-service-types';
+// These functions help build matching rules for ParseRules.
```
Comment (benjie) [inline packages/graphql-language-service-parser/src/RuleHelpers.js]: Should this comment be above the import?

Comment (benjie) [reply packages/graphql-language-service-parser/src/RuleHelpers.js]: No.

Comment (benjie) [reply packages/graphql-language-service-parser/src/RuleHelpers.js]: Ignore :)

**Review by benjie — CHANGES_REQUESTED** [review]
LGTM; just need to delete any remaining `PATENTS` files and then it's good to go up the chain for approval 👍

**Comment by acao** [comment]
@benjie thanks !! all the `PATENTS` files are gone now

**Review by benjie — APPROVED** [review]
Need approval from GraphQL Foundation; but LGTM 👍

**Comment by benjie** [comment]
@leebyron and @caniszczyk; please approve these license changes 👍

**Comment by caniszczyk** [comment]
please don't remove copyright headers, just change attribution to "GraphQL Contributors"

**Comment by acao** [comment]
@caniszczyk can do. I modelled this on Ivans PR you approved here:
 https://github.com/graphql/express-graphql/pull/515/files
Does this mean he needs to re-add the copyright headers? Or is it just different for these projects?

**Comment by acao** [comment]
@caniszczyk was easier to just re-create the branch
https://github.com/acao/graphiql/pull/9

**Pull request closed without merging**